### PR TITLE
Added Route form

### PR DESCRIPTION
**Pull request by teclator**
Added Form for adding and editing routes.

### CODE REVIEW

File: src/components/BridgeForm.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ const BridgeForm = ({ isOpen, onClose, bridge }) => {
         } else {
             setCandidatePorts(Object.values(interfaces));
         }
-    }, [interfaces]);
+    }, [bridge, isEditing, interfaces]);
```
Comment (dgdavid) [inline src/components/BridgeForm.js:52]: I'm not sure about this, but let's check it later.

File: src/components/RouteForm.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                </FormGroup>
+
+                <FormGroup
+                    label={_("Device")}
+                    isRequired
+                    fieldId="device"
+                >
+                    <FormSelect value={device} onChange={setDevice} id="device">
+                        {candidateInterfaces.map(({ name }, index) => (
+                            <FormSelectOption key={index} value={name} label={name} />
```
Comment (dgdavid) [inline src/components/RouteForm.js:135]: Just a note. According to an [article](https://medium.com/javascript-in-plain-english/you-wish-you-knew-this-little-reactjs-quirk-f021d8375dbe) that I read yesterday, maybe we should use the interface `id` as key instead the index.

**Review by dgdavid — APPROVED** [review]
Looks good! 👍

File: src/components/BridgeForm.js (hunk 1 above)
Comment (teclator) [reply src/components/BridgeForm.js:52]: eslint was complying so, the same with bonding, I'm fine discarding the change

Comment (dgdavid) [reply src/components/BridgeForm.js:52]: I know. It was just a sort of self-note to check it on a near future. Keep the change, no problem.

**Pull request merged**